src/app/components/Account/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { FC } from 'react'
+import { staking } from '@oasisprotocol/client'
```

Comment:
(depending on future uses: this is a large dependency for a small utility here. Potentially something to optimize in the future)